### ISSUE TITLE
Welcome to my new blog

### ISSUE THREAD

**Issue by rochacbruno** [rochacbruno/rochacbruno#1]
Hi there,

This is my new blog :)

Yeah you read it correcly I am using github issues as a blogging platform (well, I am trying)

## Why would someone use github issue as a blog?

- Github released the feature that enables README.md on profile pages
- I can now get a list of my latest 5 issues and list on that README :)
- I can use github actions to automate that process :+1: 

## Why not use a real blogging platform?

Here I have everything I need from a blogging platform, all the features already implemented.

And yeah I can create content locally using markdown and send to  github as a new issue using github cli.

**Comment by PotHix** [comment]
That's an interesting idea. Is this going forward somehow?

**Comment by rochacbruno** [comment]
@PotHix not yet, I still want to migrate my CMS to issues, I had some ideas but that required some Github Actions coding such as.

- Use labels to set state of posts/issues `published`, `draft` etc..
- Use projects to categorize
- Use milestones to schedule
- Render README as bog front-end
- A cleanup job to delete/lock unwanted conversations and SPAM that may happen
- a template/tool to post a `local_file.md` as a new issue/post.

That can be done in Rust/Python/Bash..

I just had no time yet to try

**Comment by PotHix** [comment]
Yes, it seems to require some effort to put everything together. If you ever do this, please ping me. I'm curious. :)

**Comment by rochacbruno** [comment]
Someone did it! @PotHix https://github.com/renatorib/github-blog

**Comment by PotHix** [comment]
That's interesting! Thanks for pinging!
